samples/js/valueentity-counter/.dockerignore

```diff
@@ -1,6 +1,4 @@
 /user-function.desc
-/docker-compose.linux.yml
-/docker-compose.yml
```

Comment:
the `docker-compose.yml` not intended to be removed (and can't suggest on deleted lines)